### PR TITLE
New leds

### PR DESCRIPTION
**Pull request by rafiaaleem5**
Created an enum and split the colors from the order logic

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -23,6 +23,9 @@
  * project.
  */
 public class Robot extends LoggedRobot {
+  private static final double lowBatteryVoltage = 9.0;
```
Comment (elleryjh) [inline src/main/java/frc/robot/Robot.java:26]: These aren't used. Delete?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -169,7 +164,7 @@ private void configureButtonBindings() {
         .pov(-1)
         .whileFalse(new DriveWithDpad(drive, () -> driverController.getHID().getPOV()));
 
-    led2023.setDefaultCommand(new LedRainbowCMD(led2023).ignoringDisable(true));
+    // led2023.setDefaultCommand(new LedRainbowCMD(led2023).ignoringDisable(true));
```
Comment (elleryjh) [inline src/main/java/frc/robot/RobotContainer.java:167]: Delete.

File: src/main/java/frc/robot/subsystems/drive/Module.java [hunk 3]
```diff
@@ -5,9 +5,10 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.kinematics.SwerveModulePosition;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import org.littletonrobotics.junction.Logger;
 
-public class Module {
+public class Module extends SubsystemBase {
```
Comment (elleryjh) [inline src/main/java/frc/robot/subsystems/drive/Module.java:11]: This is unrelated to LEDs. Revert?

File: src/main/java/frc/robot/subsystems/robotstate/RobotState.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.robotstate;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.AutoLogOutput;
+
+public class RobotState extends SubsystemBase {
```
Comment (elleryjh) [inline src/main/java/frc/robot/subsystems/robotstate/RobotState.java:6]: Please add javadoc comment.

File: vendordeps/PathplannerLib.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "fileName": "PathplannerLib.json",
   "name": "PathplannerLib",
-  "version": "2024.1.4",
+  "version": "2024.1.6",
```
Comment (elleryjh) [inline vendordeps/PathplannerLib.json:4]: Revert Pathplanner and photonlib changes

**Review by elleryjh — COMMENTED** [review]
There are format issues.

Please run the formatter by building, and then commit again.

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (rafiaaleem5) [reply src/main/java/frc/robot/Robot.java:26]: deleted

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (rafiaaleem5) [reply src/main/java/frc/robot/RobotContainer.java:167]: deleted

File: src/main/java/frc/robot/subsystems/drive/Module.java (hunk 3 above)
Comment (rafiaaleem5) [reply src/main/java/frc/robot/subsystems/drive/Module.java:11]: reverted

File: src/main/java/frc/robot/subsystems/robotstate/RobotState.java (hunk 4 above)
Comment (rafiaaleem5) [reply src/main/java/frc/robot/subsystems/robotstate/RobotState.java:6]: added javadoc comment

**Review by elleryjh — APPROVED** [review]
LGTM
